examples/service-account.go

```diff
@@ -0,0 +1,68 @@
+//
```

Comment:
You need to add this
```
 //go:build ignore
 // +build ignore
```

so this example won't be compiled with go lint and others